iio.h

```diff
@@ -199,8 +199,10 @@ enum iio_modifier {
 
 /** @brief Create a scan context
  * @param backend A NULL-terminated string containing the backend(s) to use for
- * scanning (example: pre version 0.20 :  "local", "ip", or "usb"; post version
- * 0.20 can handle multiple, including "local:usb:", "ip:usb:", "local:usb:ip:").
+ * scanning (example: post version 0.20 can handle multiple, including "local:usb:",
+ * "ip:usb:", "local:usb:ip:"; post version 0.25 usb can be limited to a specific
+ * product ID or vendor ID, by adding "pid:b673" or "vid:0456" in the same string
+ * for example : "usb,vid:0456" would limit checks on USB to vendor ID 0x0456).
```

Comment:
I think it would be better to use the equal sign (`=`) for the vid/pid parameters.

Note that in the "dev" branch with the new scan API, a comma character is used instead of the colon; so maybe we could have `usb:vid=1234&pid=5678` or `usb:vid=1234;pid=5678` to avoid confusion later.

Reply:
I was thinking it would be easier to copy things from lsusb if it used `:`, but that's sort of not true either...

from the man page:
```
 -d [vendor]:[product]
          Show only devices with the specified vendor and product ID.
          Both ID's are given in hexadecimal.
```

```
rgetz@brain:~/github/libiio$ lsusb -d 0456:
Bus 003 Device 016: ID 0456:b673 Analog Devices, Inc. 
rgetz@brain:~/github/libiio$ lsusb -d 0456:b673
Bus 003 Device 016: ID 0456:b673 Analog Devices, Inc.
```
I guess I thought that most people were already comfortable with `vid:pid` (with the colon).

but then, maybe it should be `usb=0456:b673,0456:b674` and the list continues, or multiple  `usb=0456:b673,usb=0456:b674` and `usb` (without the `=` means any).

that is sort of what dfu-utils does. (from the man page)

```
      -d, --device [Run-Time VENDOR]:[Run-Time PRODUCT][,[DFU Mode VENDOR]:[DFU Mode
       PRODUCT]]
              Specify  run-time  and/or DFU mode vendor and/or product IDs of the DFU
              device to work with. VENDOR and PRODUCT  are  hexadecimal  numbers  (no
              prefix  needed),  "*"  (match any), or "-" (match nothing). By default,
              any DFU capable device in either run-time or DFU mode will  be  considered.

              --device 1457:51ab,951:26
                     Work with a device in run-time mode with vendor  ID  0x1457  and
                     product  ID  0x51ab,  or  in  DFU mode with vendor ID 0x0951 and
                     product ID 0x0026

```

? 

I'm open to either.

Reply:
reworled